package.json

```diff
@@ -28,12 +28,14 @@
     "electron-prebuilt": "^1.2.1",
     "highlight.js": "^9.4.0",
     "json-loader": "^0.5.4",
+    "rc-collapse": "^1.6.3",
     "react": "^15.0.1",
     "react-dom": "^15.0.1",
     "react-redux": "^4.4.1",
     "react-router": "^2.4.1",
     "react-router-redux": "^4.0.2",
     "redux": "^3.3.1",
```

Comment:
I'm getting a peer-dependency conflict:
```
npm WARN redux-devtools@3.3.1 requires a peer of redux@^3.5.0 but none was installed.
```
Would you bump our redux up to 3.5.0?